framework/src/main/java/org/checkerframework/framework/type/visitor/AnnotatedTypeReplacer.java

```diff
@@ -69,14 +84,20 @@ protected Void combineRs(Void r1, Void r2) {
         return r1;
     }
 
+    /**
+     * Replace the annotations in dst with the annotations in src
+     *
+     * @param src the source of the annotations
+     * @param dst the destination of the annotations
+     */
     protected void replaceAnnotations(
-            final AnnotatedTypeMirror one, final AnnotatedTypeMirror two) {
+            final AnnotatedTypeMirror src, final AnnotatedTypeMirror dst) {
```

Comment:
It would have been more consistent to rename to `from` and `to`, like in the earlier methods.